pkg/kf/push.go

```diff
@@ -86,6 +87,14 @@ func (p *pusher) Push(appName, srcImage string, opts ...PushOption) error {
 	s.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Image = imageName
 	s.Spec.RunLatest.Configuration.RevisionTemplate.Spec.ServiceAccountName = cfg.ServiceAccount
 
+	if s.Spec.RunLatest.Configuration.RevisionTemplate.Annotations == nil {
+		s.Spec.RunLatest.Configuration.RevisionTemplate.Annotations = map[string]string{}
+	}
+
+	// The value of 0 for any of min or max means the bound is not set
+	s.Spec.RunLatest.Configuration.RevisionTemplate.Annotations[autoscaling.MinScaleAnnotationKey] = string(cfg.MinScale)
```

Comment:
are there any cases where we'd want `strconv.Itoa` here rather than string? (asking because you used it in the test).